packages/teams-js/src/public/nestedAppAuth.ts

```diff
@@ -85,7 +85,7 @@ export function canParentManageNAATrustedOrigins(): boolean {
  * @beta
  */
 export function isDeeplyNestedAuthSupported(): boolean {
-  return (ensureInitialized(runtime) && isNAAChannelRecommended() && runtime.isDeeplyNestedAuthSupported) ?? false;
+  return (ensureInitialized(runtime) && runtime.isDeeplyNestedAuthSupported) ?? false;
```

Comment:
The isNAAChannelRecommended() check is unnecessary because there won't be any case where isNAAChannelRecommended() is false but isDeeplyNestedAuthSupported() is true. It's better to keep the logic clean and simple, so the check has been removed.